src/ssl.c

```diff
@@ -11148,9 +11173,9 @@ int wolfSSL_set_compression(WOLFSSL* ssl)
             return WOLFSSL_FATAL_ERROR;
     }
 
-    int wolfSSL_OPENSSL_add_all_algorithms_noconf(void)
+    int wolfSSL_OpenSSL_add_all_algorithms_noconf(void)
```

Comment:
Because "wolfSSL_OPENSSL_add_all_algorithms_noconf()" has been a public API function, if we are going to change the name we need to add a #define mapping the old name to the new name.  This way we don't break people who have been using the old function name.

Reply:
Because function name OpenSSL_add_all_algorithms_noconf() is correct, I changed to match original OpenSSL name.Customer indicated this mistake and I modified it.
This function is implemented as new function in this project,so I guess there is no compatibility issue.
I will define alias for any reason if needed, though.

Reply:
wolfSSL_OPENSSL_add_all_algorithms_noconf(), with the capital "OPENSSL" did get released publicly in wolfSSL 3.15.3.  This means we could potentially have users actively using this function under its current name.  Let's go ahead and add a define alias for these users, just in case.  Thanks!

Reply:
Thanks!